src/strategies_and_params/poststepstrategy.jl

```diff
@@ -4,9 +4,16 @@
 Subtypes of `PostStepStrategy` can be used to perform arbitrary computation on a replica
 after an FCIQMC step is finished and report the results.
 
+# Implemented strategies:
+
+* [`ProjectedEnergy`](@ref)
+* [`Projector`](@ref)
+
```

Comment:
```suggestion
* [`SignCoherence`](@ref)
* [`WalkerLoneliness`](@ref)

```
Or did you leave them out intentionally?

Reply:
No, not intentionally. It was late last night and I didn't find them :-)